src/main/java/teammates/logic/external/LogService.java

```diff
@@ -25,8 +25,7 @@ public interface LogService {
     /**
      * Creates a feedback session log.
      */
-    void createFeedbackSessionLog(String courseId, UUID studentId, String email, UUID fsId, String fsName,
```

Comment:
nit: let's change the java doc comment here to be different from the previous, indicating that this is for the new log format for migrated courses, something like: Creates a feedback session log for courses migrated to SQL